cookbooks/bach_sqoop/README.md

```diff
@@ -0,0 +1 @@
+Cookbook to install Apache Sqoop on data nodes.
```

Comment:
Could you please provide a more full README? Describing the attributes and recipes available?